zip-1014.rst

```diff
@@ -61,19 +66,24 @@ Motivation
 
 Starting at Zcash's first halving in October 2020, by default 100% of the block
 subsidies will be allocated to miners, and no further funds will be automatically
-allocated to research, development, and outreach. Consequently, no substantial
-new funding may be available to existing teams dedicated to Zcash: the Electric
-Coin Company (ECC), the Zcash Foundation (ZF), and the many entities funded by
-the ZF grant program.
+allocated for charitable, educational, or scientific purposes, such as  research,
+development, and outreach. Consequently, no substantial
+new funding may be available to existing teams dedicated to furthering such charitable
+or educational purposes: the Electric Coin Company (ECC), the Zcash Foundation (ZF), and
+the many entities funded by the ZF grant program.
```

Comment:
This is referring to what *would* have happened by default if the Dev Fund had not been agreed. Therefore this change should be reverted, I think.

Reply:
I agree, although I do not think reverting it is _required_. I believe the new edit only qualifies that R&D and outreach are "charitable, educational, or scientific purposes", right?

Reply:
By saying "no further funds", it seems to me to be suggesting that the previous funds from the Founders' Reward were all directed to charitable or educational purposes. Some of them were, but they weren't required to be, and it's unnecessary to say or imply that they were.